### DISCUSSION TITLE
guides/model-evaluation-insights/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#14947] [Docs]
# guides/model-evaluation-insights/

Explore the most effective ways to assess and refine YOLOv8 models for better performance. Learn about evaluation metrics, fine-tuning processes, and how to customize your model for specific needs.

https://docs.ultralytics.com/guides/model-evaluation-insights/

**Comment by ILIAHHne63** [comment]
Hello, I'm using YoLOv10, how can I load the pretrained backbone, but leave the head unpretrained?

**Reply by glenn-jocher** [reply]
@ILIAHHne63 to load a pretrained backbone while leaving the head unpretrained in YOLOv10, you can initialize the model with the pretrained weights and then manually reset the head layers. This allows you to retain the pretrained features of the backbone while training the head from scratch. For detailed steps, refer to the model fine-tuning section in our documentation. If you encounter any issues, ensure you're using the latest version of the package.
